.github/workflows/mako.yml

```
@@ -0,0 +1,37 @@
+name: mako-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+    mako-testsuite-1_1_4:
+    runs-on: ubuntu-latest
+    env:
+      DD_TESTING_RAISE: true
+    defaults:
+      run:
+        working-directory: mako
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+        with:
+          repository: sqlalchemy/mako
+          ref: 1.1.4
+          path: mako
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install tox
+        run: pip install tox
+      - name: Create tox env
+        run: tox -e py38 --notest
```

The mako `tox.ini` only has 1 environment which is just called `py`

```suggestion
        run: tox -e py --notest
```